Provide an API for applications to be able to transform the return values from their `source` function before rendering the suggestions. (#12)
Financial-Times/o-autocomplete: README.md

| Name | Type | Description |
| --- | --- | --- |
| options | <code>Array.&lt;*&gt;</code> | The options which match the entered query |

### mapOptionToSuggestedValue

> This function is used to convert the options returned from the `source` function into a their respective strings to be used as the suggestion values within the suggestions menu.

Reviewer: Started this suggestion to correct the "into a their" typo, but think it can be simplified a bit, what do you think?
```suggestion
This function is used to convert the options returned from the `source` function into strings for the suggestions menu.
```

Author: I like it 👍
